frontend/src/pages/Rubrics/RubricEditor/index.tsx

```diff
@@ -0,0 +1,215 @@
+import { useContext, useEffect, useState } from "react";
+
+import "./styles.css";
+import Button from "@/components/Button";
+import Input from "@/components/Input";
+import RubricItem from "@/components/RubricItem";
+import { createRubric, updateRubric } from "@/api/rubrics";
```

Comment:
Add a deleteRubric function and functionality to edit an existing rubric line item! TYYY